app/models/school.rb

```diff
@@ -205,4 +205,18 @@ def slug_candidates
   def cip_only_establishment_type?
     CIP_ONLY_TYPE_CODES.include?(school_type_code)
   end
+
+  def active_profiles_for(cohort:, type: ParticipantProfile)
+    school_cohort = school_cohorts.find_by(cohort:)
```

Comment:
discussed in person: do we want the latest school (latest Induction record) vs if they ever were with that school?